ivy/functional/frontends/torch/nn/functional/vision_functions.py

```diff
@@ -120,3 +120,108 @@ def pad(input, pad, mode="constant", value=0):
                 + "['constant', 'reflect', 'replicate', 'circular']"
             ).format(mode)
         )
+
+
+@to_ivy_arrays_and_back
+def upsample_bilinear(input, size=None, scale_factor=None):
+    if scale_factor and size:
+        raise ivy.exceptions.IvyException(
```

Comment:
Given that the function should only take 4-dimensional inputs, another exception could be introduced to handle inputs with dimensions different from 4

Reply:
Thank you for review. In the original library, this exception is thrown by lower level function torch.nn.functional.interpolate function. Should I still handle even if in the original library it is handled by interpolate (which is not implemented yet in ivy torch frontend)?

Reply:
Hi, 4D input handling should be finished : 

[783cbc8](https://github.com/unifyai/ivy/pull/7338/commits/783cbc8fce57536b757a066d4e3a8cbaa672b815)
[9a9513c](https://github.com/unifyai/ivy/pull/7338/commits/9a9513cf6f24fa4b62d326e0e439820f72b04967)